waku/standards/core/64/network.md

```diff
@@ -263,16 +256,16 @@ the relay node MUST _reject_ the message.
 
 If a message contains an RLN proof
 and the zero-knowledge proof is invalid
-according to the verification process described in [32/RLN-V1](../../../../vac/32/rln-v1.md),
+according to the verification process described in [32/RLN-V1](../../../../vac/32/rln-v1.md)/[RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md?plain=1),
```

Comment:
Same question as before - shouldn't we rather drop all references to RLN-V1?

Reply:
dropped the reference: fd45bb7495c9123dd2f00becf30643db5660e6e3